aiida/cmdline/commands/cmd_status.py

```diff
@@ -14,13 +14,26 @@
 
 import sys
 
+import logging
 from enum import IntEnum
+from contextlib import contextmanager
 import click
 
 from aiida.cmdline.commands.cmd_verdi import verdi
 from ..utils.echo import ExitCode
 
 
+@contextmanager
+def set_log_level(logger, level):
```

Comment:
I think this utility function might be nice to put in `aiida.common.log` so it can be reused. Maybe good to check the code if something similar is already done and replace with this utility

Reply:
I checked that module and didn't see anything similar, but I'll check again and make an assessment whether this is worth generalizing.

Reply:
I searched the code and found that many tests use the following pattern:
```
logging.disable(level=logging.CRITICAL)
# noisy code
logging.disable(level=logging.NOTSET)
```
I think that's fine as long as it's properly placed in `try: ... finally:` blocks and has the added benefit that it suppresses logging from **all** modules.

I'll adjust this patch accordingly.

Reply:
I liked your context manager approach a lot better though. What I suggest is that you place that in `aiida.common.log` and have it just disable logging completely for its duration, just as you have done now locally in the status command. This utility can then replace the same pattern you found in the other tests.